charts/judge/README.md

```diff
@@ -0,0 +1,56 @@
+# JUDGE Helm Chart
```

Comment:
I think this should move to the root of the repo and the repo should default to being _the_ location for the Judge chart.